### PR TITLE
Feature: Add filters for user info field values

### PR DESCRIPTION
**Pull request by kjohnson**
## Description

This PR adds filters for the prefill form field values.

- `give_user_info_fields_user_info`
- `give_default_cc_address_fields_user_info`

Shout-out to @pulsovi for the original PR.

#### Todo
- [ ] Mention @pulsovi in the squash commit.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205663032594581

### CODE REVIEW

**Review by pauloiankoski — APPROVED** [review]
It does look good to me!

**Comment by pulsovi** [comment]
Hello,

Please add `$form_id` as hook parameter. I really need it for my use case.

File: includes/forms/template.php [hunk 1]
```diff
@@ -752,7 +752,7 @@ function give_add_button_open_form( $form_id, $args ) {
 function give_user_info_fields( $form_id ) {
 
 	// Get user info.
-	$give_user_info = _give_get_prefill_form_field_values( $form_id );
+	$give_user_info = apply_filters('give_user_info_fields_user_info', _give_get_prefill_form_field_values( $form_id ));
```
[the remark this answers is not in the document]
Comment (kjohnson) [reply includes/forms/template.php]: Resolved, see [4e50474](https://github.com/impress-org/givewp/pull/6989/commits/4e5047474d7d6ab5354dbc58f273f1df4aaf6c5e).

File: includes/forms/template.php [hunk 2]
```diff
@@ -1203,7 +1203,7 @@ class="card-expiry give-input required" placeholder="<?php esc_attr_e( 'MM / YY'
 function give_default_cc_address_fields($form_id, $return = false)
```
[the remark this answers is not in the document]
Comment (kjohnson) [reply includes/forms/template.php:1206]: Resolved, see [4e50474](https://github.com/impress-org/givewp/pull/6989/commits/4e5047474d7d6ab5354dbc58f273f1df4aaf6c5e).

**Comment by kjohnson** [comment]
@glaubersilva thanks for reminding me about the unreleased tag!

@pulsovi I added `$form_id` as an argument for both filters.

**Review by glaubersilva — APPROVED** [review]
@kjohnson Thanks for the changes, looks good to me.

**Comment by kjohnson** [comment]
Building for QA, see https://github.com/impress-org/givewp/actions/runs/6424327474.

**Review by rickalday — APPROVED** [review]
Passed manual QA tests

**Comment by pulsovi** [comment]
Hello,
Thanks for the work on this !

What is the first version of GiveWP that will include this new feature?

**Pull request merged**